frontend/src/components/payment/paypal-payment-dialog.js

```diff
@@ -52,7 +58,8 @@ class PaypalPaymentDialog extends Component {
               marginTop: 40,
               fontFamily: 'Roboto',
               background: '#ecf0f1',
-              padding: '20px' } }>
+              padding: '20px'
```

Comment:
You can just use numbers here:
```suggestion
              padding: 20
```

Reply:
Done :sunglasses: